MainController.php

```diff
@@ -1,8 +1,37 @@
 <?php
 
-function main() {
-  // function to create a new file
-  // the file should have a unique name, and the content of the function is the current timestamp (UNIX format).
+declare(strict_types=1);
+function main(string $fileDirectory = null, string $content = null)
+{
+    // function to create a new file
+    // the file should have a unique name, and the content of the function is the current timestamp (UNIX format).
+    try {
+        $filename = "file_" . time() . ".txt";
```

Comment:
I meant it the other way round - time is the content, filename is a unique string